ffc-helm-library/templates/_azure-identity.yaml

```diff
@@ -0,0 +1,15 @@
+{{- define "ffc-helm-library.azure-identity.tpl" -}}
+{{- $requiredMsg := include "ffc-helm-library.default-check-required-msg" . -}}
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: {{ required (printf $requiredMsg "azureIdentity.name") .Values.azureIdentity.name | quote }}
+  namespace: {{ required (printf $requiredMsg "namespace") .Values.namespace | quote }}
```

Comment:
Labels should also be attached to this resource

Reply:
added